src/logtab.cpp

```diff
@@ -96,8 +96,34 @@ void LogTab::InitMenus()
     m_hideSelectedType = new QAction(QIcon(GetThemedIcon(":/ctx-hide.png")), "Hide all events of this type", this);
     connect(m_hideSelectedType, &QAction::triggered, this, &LogTab::RowHideSelectedType);
 
+    // Top-level `Export events`
     m_exportToTabAction = new QAction(QIcon(GetThemedIcon(":/ctx-newtab.png")), "Export event(s) to new tab", this);
-    connect(m_exportToTabAction, &QAction::triggered, this, &LogTab::ExportToNewTab);
+    connect(m_exportToTabAction, &QAction::triggered, [this]() { ExportToNewTab(); });
+    // Populate `Export to tab` menu
+    m_exportToTabMenu = new QMenu("Export to new tab",this);
+    m_exportToTabMenu->setIcon(QIcon(GetThemedIcon(":/ctx-newtab.png")));
+    {
+        // This thread
+        QAction* exportSelectedType = new QAction(QIcon(GetThemedIcon(":/ctx-newtab.png")), "this type", m_exportToTabMenu);
```

Comment:
minor: is "this" the first word appearing in the menu option? should it be capitalized ("This")?
This applies to all actions

Reply:
Fixed for both "Export" and "Highlight"